gcc_struct attribute directive ignored warning in AArch64 builds

The AArch64 build throws up 267 warnings in the elfutils submodule to do with lack of GCC compiler support for `__attribute__ ((packed, gcc_struct))`, e.g.
```
In file included from /work/sources/dynamorio/third_party/elfutils/libdwelf/libdwelfP.h:33,
                 from /work/sources/dynamorio/third_party/elfutils/libdwelf/dwelf_dwarf_gnu_debugaltlink.c:33:
/work/sources/dynamorio/third_party/elfutils/libdw/libdwP.h:279:1: warning: ‘gcc_struct’ attribute directive ignored [-Wattributes]
  279 | } attribute_packed;
      | ^
```
The `gcc_struct` attribute modifier in `ext/drsyms/elfutils/config.h` appears to be selectable using `HAVE_GCC_STRUCT`:
```
#ifdef HAVE_GCC_STRUCT
#define attribute_packed \
  __attribute__ ((packed, gcc_struct))
#else
#define attribute_packed \
  __attribute__ ((packed))
#endif
```
Is the warning telling us something we need to action or a cautionary message we can ignore?
Removing it on AArch64 makes no difference to test results.

--- a/ext/drsyms/elfutils/config.h
+++ b/ext/drsyms/elfutils/config.h
@@ -72,7 +72,9 @@
 #define HAVE_FALLTHROUGH 1
 
 /* Defined if __attribute__((gcc_struct)) is supported */
-#define HAVE_GCC_STRUCT 1
+#if defined(__x86_64__) || defined(__i386__)
+#    define HAVE_GCC_STRUCT 1
+#endif
 
 /* Define to 1 if you have the `getrlimit' function. */
 #define HAVE_GETRLIMIT 1

